packages/react/counter/src/Counter.tsx

```diff
@@ -118,18 +136,19 @@ export function Counter(props: CounterProps) {
     async function setup() {
       if (isStatic && value) {
         setDataValue(value)
-        return
       }
 
-      const fetchedValue = await fetchData()
+      if (!isStatic) {
+        const fetchedValue = await fetchData()
 
-      if (!fetchedValue) {
-        setHasError(true)
-        console.error(`QueryError: Your metric ${query?.metric} returned undefined or a \`null\` value.`)
-        return
-      }
+        if (typeof fetchedValue === 'undefined') {
+          setHasError(true)
+          console.error(`QueryError: Your metric ${query?.metric} returned undefined.`)
```

Comment:
@markandrus I think we should put in discussion a better way to structure error messages. @markandrus do you have any suggestion we could do on a separate PR?